### PR TITLE
Update README.md about travis-sphinx

### PR DESCRIPTION
**Pull request by ShigekiKarita**
for #58

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -2,44 +2,25 @@
 
 ## Install
 
+We use [travis-sphinx](https://github.com/Syntaf/travis-sphinx) to generate & deploy HTML documentation.
+
 ```sh
 $ pip install sphinx
 $ pip install sphinx_rtd_theme
 $ pip install commonmark==0.5.4 recommonmark
+$ pip install travis-sphinx
```
Comment (sw005320) [inline doc/README.md]: How about preparing requirements.txt?

Comment (ShigekiKarita) [reply doc/README.md]: That's nice

File: doc/conf.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../src/'))
```
Comment (ShigekiKarita) [inline doc/conf.py]: I changed this path to `import e2e_asr_common` in `src/nets/*.py` and to build doc properly.

- Travis-CI failures in current master https://travis-ci.org/espnet/espnet/jobs/325498419#L775
- generated master doc https://espnet.github.io/espnet/apis/nets.html
- Travis-CI success in this PR https://travis-ci.org/ShigekiKarita/espnet/jobs/325531968#L747
- generated this PR doc https://shigekikarita.github.io/espnet/apis/nets.html

File: doc/conf.py [hunk 3]
```diff
@@ -18,7 +18,7 @@
...
+sys.path.insert(0, os.path.abspath('../src/nets'))
```
Comment (ShigekiKarita) [inline doc/conf.py]: I changed this path to `import e2e_asr_common` in `src/nets/*.py` and to build doc properly.

- Travis-CI failures in current master https://travis-ci.org/espnet/espnet/jobs/325498419#L775
- generated master doc that shows nothing but `nets.e2e_asr_common.end_detect` https://espnet.github.io/espnet/apis/nets.html
- Travis-CI success in this PR https://travis-ci.org/ShigekiKarita/espnet/jobs/325531968#L747
- generated this PR doc https://shigekikarita.github.io/espnet/apis/nets.html

**Comment by sw005320** [comment]
LGTM

**Pull request merged**